lib/importers/tpas_satisfaction/saver.rb

```diff
@@ -0,0 +1,28 @@
+module Importers
+  module TpasSatisfaction
+    class Saver
+      def initialize(records:)
+        @records = records.select(&:valid?)
+      end
+
+      def save
+        ActiveRecord::Base.transaction do
+          satisfaction!
+        end
+      rescue CodeLookup::MissingMappingError => e
+        Bugsnag.notify(e)
```

Comment:
Same with this `Bugsnag` dependency, perhaps we should inject as a matter of course?